topology/src/main/java/indexingTopology/topology/kingbase/KingBaseTopology.java

```diff
@@ -71,7 +71,7 @@
      * ingest api configuration
      */
     @Option(name = "--ingest-server-ip", usage = "the ingestion server ip")
-    private String IngestServerIp = "localhost";
+    private String IngestServerIp = "172.31.6.161";
```

Comment:
Don't change the default setting. The intend of the default "localhost" is to enable the topology work in the local mode with the default settings.